homeassistant/components/moehlenhoff_alpha2/manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "Möhlenhoff Alpha 2",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/moehlenhoff_alpha2",
-  "requirements": ["moehlenhoff-alpha2==1.2.1"],
+  "requirements": ["moehlenhoff-alpha2==1.3.0"],
```

Comment:
Please rebase on latest dev branch as this change is now merged.

Reply:
Changes are rebased.